app/components/documents/document-badge.hbs

```diff
@@ -3,6 +3,9 @@
     href={{@doc.viewDocumentURL}} rel="noreferrer noopener"
     data-test-document-badge-link
   >
+    {{#if @doc.documentContainer}}
+      {{await @doc.documentContainer.position}}.&nbsp;
+    {{/if}}
```

Comment:
containers without position will still show the dot.
![image](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/assets/22245223/96db691a-2acb-4e78-9452-a38459ac42ab)